engine/src/dskw32.cpp

```diff
@@ -1449,6 +1553,13 @@ struct MCStdioFileHandle: public MCSystemFileHandle
 
 	virtual uint64_t GetFileSize(void)
 	{
+		/* First flush any write buffer, if this fails, then the operation
+		 * fails. */
+		if (!FlushWriteBuffer())
+		{
+			return false;
```

Comment:
returning a boolean as an int64_t?

Reply:
Similar to the Tell issue - either it needs to return an error (and the size in an out) *or* it just synthesises the value.